packages/device_info/device_info_platform_interface/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
-## 2.0.0-nullsafety.2
+## 2.0.0
 
 * Make `baseOS`, `previewSdkInt`, and `securityPatch` nullable types.
 * Remove default values for non-nullable types.
-
-## 2.0.0-nullsafety.1
-
-* Bump Dart SDK to support null safety.
-
-## 2.0.0-nullsafety
-
 * Migrate to null safety.
```

Comment:
Nit: let's make this first since it's the key change in the 2.0.0 release.

Reply:
done